### ISSUE TITLE
List Namecoin Implementations

### ISSUE THREAD

**Issue by JeremyRand** [namecoin/namecoin.info#95]
Something like this: https://docs.google.com/document/d/1n2-gN4WDngy9aJDG9cpjfx27eVkuO86kywJqBQ3Y9mw/edit?usp=sharing

Any comments?

**Comment by phelixbtc** [comment]
Looks good. I suggest putting it on the wiki, though. With a link to it that is. Maybe we should create a "ressources" page or something.

**Comment by JeremyRand** [comment]
Linking to end-user software should not be on the wiki -- some level of review should be required before edits are made, which the wiki cannot accomplish.

**Comment by phelixbtc** [comment]
Are you sure it is not possible to have wiki pages reviewed before publishing changes? Administering the list is way more tedious in html, also it is rare for outsiders to suggest PRs.

**Comment by JeremyRand** [comment]
https://www.mediawiki.org/wiki/Extension:FlaggedRevs can allow review by 1-of-N moderators, but not M-of-N moderators.  I'd prefer requiring 2 devs to approve edits if possible.  Am I being overly paranoid?

**Comment by phelixbtc** [comment]
We are thin stretched with resources, 1-of-N has to suffice. Also it is ok from my point of view. Now only somebody needs to install that extension...

**Comment by JeremyRand** [comment]
I think it's already installed, but misconfigured (it's possible to mark pages as reviewed, but the unreviewed pages are still showing up to end users).  I don't know how to fix that.  @indolering was working with that, I think?

**Comment by indolering** [comment]
I agree with Phelix that 1-of-n mods is enough and messing with the flagged revisions extension isn't worth the effort.  I believe that it is only a visual distinction (which isn't good enough) and there aren't enough mods to keep up with the proposed changes.

Furthermore, this is mainly information for developers, we should have a list of clients on the website akin to Bitcoin's.

**Comment by JeremyRand** [comment]
@indolering The list of implementations is not primarily for developers.  The whole point of this ticket was to have it on the website, like Bitcoin.  What do you mean by "only a visual distinction"?  The default should be that *anything* aimed at end users needs moderator approval before being visible to end users.

**Comment by JeremyRand** [comment]
If setting up the extension is too much effort for people, I volunteer to try setting it up; just make my account (biolizard89) an administrator and I'll do it.

**Comment by indolering** [comment]
> The list of implementations is not primarily for developers. The whole point of this ticket was to have it on the website, like Bitcoin. What do you mean by "only a visual distinction"? The default should be that anything aimed at end users needs moderator approval before being visible to end users.

FWIW, Jeremy and I discussed this on IRC and we both agreed that Namecoin has another set of advanced users and system admins with needs that are distinct from that of developers, domain owners, and traditional end users.  This needs it's own distinct documentation and the wiki is not the ideal place for it but Jeremy has an idea for shoehorning it in for now.

@JeremyRand Flagged revisions is already installed on the wiki. I believe you have administrative privileges on the wiki.  I can create a user account for you this weekend if you need it.
